lib/value-pairs/cmdline.c

```diff
@@ -439,6 +441,10 @@ value_pairs_new_from_cmdline (GlobalConfig *cfg,
       "subkeys", 0, 0, G_OPTION_ARG_CALLBACK, vp_cmdline_parse_subkeys,
       NULL, NULL
     },
+    {
+      "omit-empty-values", 0, 0, G_OPTION_ARG_NONE, &vp->omit_empty_values,
```

Comment:
I would rather create an `value_pairs_omit_empty()` and call it from here (so the struct does not need to be moved to the header).

Reply:
> I would rather create an `value_pairs_omit_empty()` and call it from here (so the struct does not need to be moved to the header).

It sounds good.
@bazsi Can I update this branch with the change requested by @Kokan?

Reply:
I went ahead and updated the branch. Let me know if you don't like it.